src/NavConfig.jsonc

```diff
@@ -49,6 +49,7 @@
     {
       "identity": "QuietBackgroundProcessesExperiment",
       "enabledByDefault": false,
+      "isFeaturePresentCheck": true,

```

Comment:
Please update https://github.com/microsoft/devhome/blob/main/docs/tools/navconfig.md

Reply:
What if we call this "NeedsFeaturePresenceCheck"? I think that may be a little clearer, especially thinking abou tthis line
```ExperimentalFeatures = experimentationService!.ExperimentalFeatures.Where(x => x.IsVisible && (!x.IsFeaturePresentCheck || IsFeaturePresent(x))).OrderBy(x => x.Id).ToList();```

Reply:
navconfig.md is too out of date w.r.t. experimentalFeatures.